bootstrap.sh

```diff
@@ -49,20 +49,20 @@ function create_standalone_venv {
 
 	# from http://stackoverflow.com/questions/4324558/whats-the-proper-way-to-install-pip-virtualenv-and-distribute-for-python
 	# Select current version of virtualenv:
-	VERSION=1.8.2
-	URL_BASE="http://pypi.python.org/packages/source/v/virtualenv"
+	VERSION=15.0.2
+	URL_BASE="https://pypi.python.org/packages/5c/79/5dae7494b9f5ed061cff9a8ab8d6e1f02db352f3facf907d9eb614fb80e9/virtualenv-15.0.2.tar.gz#md5=0ed59863994daf1292827ffdbba80a63"
```

Comment:
This method is not documented, it relies on direct link from https://pypi.python.org/pypi/virtualenv#downloads. I did that because downloading 15.0.2 by official link is broken: https://pypi.python.org/packages/source/v/virtualenv/virtualenv-15.0.2.tar.gz, and using older version (like 1.8.2) was causing some problems on my machine. So I proposed this change for you to review. Sorry for not documenting my reasons earlier. My commit may be a temporary fix only and very specific one, so you should probably carefully review - and maybe discard - it.